examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -182,15 +185,109 @@
 # Grab parcellations of brain image stored in attribute `labels_img_`
 kmeans_labels_img = kmeans.labels_img_
 
-plotting.plot_roi(kmeans_labels_img, mean_func_img,
-                  title="KMeans parcellation",
-                  display_mode='xz')
+display = plotting.plot_roi(kmeans_labels_img, mean_func_img,
```

Comment:
```suggestion
plotting.plot_roi(kmeans_labels_img, mean_func_img,
```

Reply:
actually this is used to reuse cut_coords in the next figure so this is necessary.